py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -80,23 +81,45 @@ def index(
     source_ir: Optional[SourceIR],
     name: str,
     input: TRTTensor,
-    index: Union[TRTTensor, Sequence[TRTTensor]],
+    index: Union[
+        TRTTensor,
+        Sequence[TRTTensor],
+        np.ndarray,
+        Sequence[np.ndarray],
+        torch.Tensor,
+        Sequence[torch.Tensor],
+    ],
 ) -> TRTTensor:
     adv_indx_indices = []
     tensor_indices = []
     # _LOGGER.debug(f"The index shape is {index.shape}")
     # check if the input is dynamic
     dynamic_shape = has_dynamic_shape(input.shape)
-
+    # is_numpy is a flag to specify if all the indices are numpy or torchTensor.
+    # If any is not this flag will be set to False
+    is_numpy = True
+    _LOGGER.debug(f"Checking for the is_numpy flag")
```

Comment:
Consider switching to `"Determining whether aten.index constant-index optimization can be invoked"`